device-registry/src/test/scala/org/genivi/sota/device_registry/DeviceRequests.scala

```diff
@@ -42,8 +42,10 @@ trait DeviceRequests { self: ResourceSpec =>
   def fetchDevice(uuid: Uuid): HttpRequest =
     Get(Resource.uri(api, uuid.show))
 
-  def searchDevice(namespace: Namespace, regex: String): HttpRequest =
-    Get(Resource.uri(api).withQuery(Query("namespace" -> namespace.get, "regex" -> regex)))
+  def searchDevice(namespace: Namespace, regex: String, offset: Long = 0, limit: Long = 50)
+      :HttpRequest =
+    Get(Resource.uri(api).withQuery(Query("namespace" -> namespace.get, "regex" -> regex,
+                                          "offset" -> offset.toString, "limit" -> limit.toString)))
```

Comment:
I know this was wrong before but the search endpoint doesn't actually take a `namespace` via a query parameter, but it does take a `deviceId`